docs/spfx/library-component-tutorial.md

```diff
@@ -45,7 +45,7 @@ localization_priority: Priority
  
 1. Open the solution in your favorite editor and navigate to ```src\corporateLibrary\CorporateLibraryLibrary.ts```
```

Comment:
Inline code references should be surrounded with a single backtick, not triple. Triple backticks are used to fence code blocks, not inline code.

Reply:
Shouldn't it be bold cause it is a reference to a file an not per se code?

Reply:
Correct... should be bold.